build.gradle

```diff
@@ -128,8 +166,50 @@ dependencies {
     testImplementation "org.apache.derby:derby:10.11.1.1"
     testRuntime "org.slf4j:slf4j-log4j12:$slf4jVersion"
 
+    integrationTestImplementation "org.apache.kafka:connect-api:$kafkaVersion"
+    integrationTestImplementation "org.apache.kafka:connect-runtime:$kafkaVersion"
+    integrationTestImplementation "org.apache.kafka:connect-json:$kafkaVersion"
+    integrationTestImplementation "org.apache.kafka:connect-transforms:$kafkaVersion"
+    integrationTestImplementation "io.confluent:kafka-avro-serializer:$confluentPlatformVersion"
+    integrationTestImplementation "io.confluent:kafka-connect-avro-converter:$confluentPlatformVersion"
+    integrationTestImplementation "org.apache.avro:avro:1.8.1"
+
+    integrationTestImplementation 'org.junit.jupiter:junit-jupiter:5.7.1'
+    integrationTestRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.7.1'
+
+    integrationTestImplementation "org.testcontainers:junit-jupiter:1.14.3"
+    integrationTestImplementation "org.testcontainers:testcontainers:1.14.3"
+    integrationTestImplementation "org.testcontainers:kafka:1.14.3"
+    integrationTestImplementation "org.testcontainers:postgresql:1.14.3"
+
+    integrationTestImplementation "cloud.localstack:localstack-utils:0.2.5"
+
+    integrationTestImplementation sourceSets.test.output
+}
+
+task integrationTest(type: Test) {
+    description = 'Runs the integration tests.'
+    group = 'verification'
+    testClassesDirs = sourceSets.integrationTest.output.classesDirs
+    classpath = sourceSets.integrationTest.runtimeClasspath
+
+    dependsOn test, distTar
+
+    useJUnitPlatform()
+
+    // Run always.
+    outputs.upToDateWhen { false }
+
+    if (project.hasProperty("tablePollIntervalMs")) {
+        //pass table.poll.interval.ms
+        systemProperty("integration-test.table.poll.interval.ms", project.hasProperty("tablePollIntervalMs"))
+    }
```

Comment:
I'm not sure it's a good idea to put the burden of choosing this value (or even knowing the value must be chosen) on whoever is going to run the tests. For example, I frankly have no idea what I should insert here without some research. We just need one good enough value set as a constant in tests somewhere.